src/app/components/ErrorFormatter/index.tsx

```diff
@@ -27,20 +28,34 @@ export function ErrorFormatter(props: Props) {
     [WalletErrors.InvalidNonce]: t('errors.invalidNonce'),
     [WalletErrors.DuplicateTransaction]: t('errors.duplicateTransaction'),
     [WalletErrors.NoOpenWallet]: t('errors.noOpenWallet'),
-    [WalletErrors.USBTransportError]: t('errors.usbTransportError', 'USB Transport error : {{message}}', {
-      message,
-    }),
+    [WalletErrors.USBTransportError]: t(
+      'errors.usbTransportError',
+      'USB Transport error : {{message}}. This usually means your browser is not supported - Chrome should work, but Firefox does not support WebUSB.',
+      {
+        message,
+      },
+    ),
     [WalletErrors.LedgerAppVersionNotSupported]: t(
       'errors.ledgerAppVersionNotSupported',
-      'Received : "Ledger App Version not supported", this can be a false positive, retry the operation, if the error persists, update the Oasis App on your Ledger.',
+      'Could not open Oasis App on Ledger, make sure it is unlocked and that the Oasis App is opened, and up to date.',
```

Comment:
```suggestion
      'Could not open Oasis App on Ledger. Make sure Ledger is unlocked, the Oasis App is opened and up to date.',
```

Reply:
but it really might be an unsupported version, right? should we keep something about the original message too?

Reply:
I don't know how to install an old version to check what it throws in that case

Reply:
How about:
```diff
- Could not open Oasis App on Ledger. Make..
+ Oasis App on Ledger is closed or outdated. Make..
```

Reply:
ok